foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,54 @@
 ### Introduction
 
-We're going to make a simple implementation of grade-school classic "rock paper scissors". If you don't know what that is check the [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or [this](https://www.wikihow.com/Play-Rock,-Paper,-Scissors) detailed step-by-step. For the moment we're just going to play the game from the browser console, but __we will revisit this project in a later lesson and add a Graphical User Interface with buttons and text,__ so don't forget to keep the code on GitHub! You might notice some 'Live Preview' links in the student solutions that have a GUI - this is coming in a later lesson. When you get there don't forget to come back and add your link!
+For this project, you'll create a console-only version of the grade-school game: **Rock Paper Scissors**. To learn more, or if you're unfamiliar with the game, read this [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or this [detailed step-by-step guide](https://www.wikihow.com/Play-Rock,-Paper,-Scissors)
 
-### Quick exercises before starting
+<div class="lesson-note">
+<h4>Student solutions</h4>
+Some students' solutions below include buttons, text, and other elements. Those elements form what is known as a Graphical User Interface (GUI). You'll add those elements—the GUI, in a later lesson. So remember to push your code to GitHub.
+</div>
+
+### Quick exercise before starting
+1. Identify three ways to include JavaScript in a page.
+2. Check if the page includes JavaScript:
+   - Write `console.log("Hello World");` in JavaScript.
+   - Check if `console.log("Hello World");` logs in the browser's console.
 
-1.  Identify three ways to include JavaScript in a page.
-2.  Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browser's console.
+### Problem solving approach
 
-Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving. Plan your solution out before writing any code, and test each piece as you build to ensure it is working before moving on to the next!
+Remember the [previous lesson on problem solving](https://www.theodinproject.com/lessons/foundations-problem-solving) before diving into the project. Plan your solution—write the code—and then ensure the code works.
```

Comment:
I am not fond of the redundancy of:
>Since this is your first JavaScript program built from scratch

I believe it serves no purpose other than adding more text to read. The learner already knows that this is their first program (if it is). There is no point in telling them.